activiti-cloud-modeling/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 authorizations.security-constraints[0].authRoles[0]=${ACT_KEYCLOAK_MODELER_ROLE:ACTIVITI_MODELER}
 authorizations.security-constraints[0].securityCollections[0].patterns[0]=${ACT_KEYCLOAK_PATTERNS:/v1/*}
+authorizations.security-constraints[1].securityCollections[0].patterns[0]=/v1/identity/*
```

Comment:
is this the config that we need to change once the FE stops using the old API?

Reply:
yes